codebuild/test_vectors/test_publish.yml

```diff
@@ -0,0 +1,26 @@
+version: 0.2
+
+env:
+  variables:
+    NODE_OPTIONS: "--max-old-space-size=4096"
+    NPM_CONFIG_UNSAFE_PERM: true
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: latest
+    commands:
+      - npm ci
+  build:
+    commands:
+      - npm test
+      - |-
+        if [ "$PUBLISH_LOCAL" = "true" ]; then
```

Comment:
When would we want to run `PUBLISH_LOCAL == true` and `CI == false`?
If they should always go tother then they should be a single variable.